src/gui/styles/colors.js

```diff
@@ -0,0 +1,54 @@
+import { css } from 'glamor'
+
+// Choose color : https://material.io/color/
+
+const DEFAULT = '#fff'
+const DEFAULT_TEXT = '#000'
+
+const PRIMARY = '#37474f'
+const PRIMARY_LIGHT = '#62727b'
+const PRIMARY_DARK = '#102027'
+const PRIMARY_TEXT = '#fff'
+
+const SECONDARY = '#f57f17'
+const SECONDARY_LIGHT = '#ffb04c'
+const SECONDARY_DARK = '#bc5100'
+const SECONDARY_TEXT = '#fff'
+
+// default
+export const defaultColor = css({
+  backgroundColor: DEFAULT,
+  color: DEFAULT_TEXT,
+})
+
+// primaries
+export const primaryColor = css({
+  backgroundColor: PRIMARY,
+  color: PRIMARY_TEXT,
+})
```

Comment:
That's not the best way to define colors, I think we should make it more atomic : 
`const primaryFontColor = css(...)`
`const primaryBgColor = css(...)`
...